test/common_mark/block_quotes.unit

```diff
@@ -103,10 +103,9 @@ foo
     - bar
 <<<
 <blockquote>
-<p>foo</p>
```

Comment:
Did you manually make these changes? These files are generated from Commonmark's specs, I believe currently from version 0.29.

Reply:
Yes, I did it manually. So I will figure it out how to generate these files later.

I thought it has been updated to 0.30 because of this PR: https://github.com/dart-lang/markdown/pull/383

But both 0.29 and 0.30 have the same way to parser this case:

https://spec.commonmark.org/0.29/#example-208
https://spec.commonmark.org/0.30/#example-238